pkg/injector/pod_patch.go

```diff
@@ -431,7 +431,20 @@ func getKubernetesDNS(name, namespace string) string {
 	return fmt.Sprintf("%s.%s.svc.cluster.local", name, namespace)
 }
 
-func getSidecarContainer(annotations map[string]string, id, daprSidecarImage, namespace, controlPlaneAddress, placementServiceAddress string, tokenVolumeMount *corev1.VolumeMount, trustAnchors, certChain, certKey, sentryAddress string, mtlsEnabled bool, identity string) (*corev1.Container, error) {
+func getPullPolicy(pullPolicy string) corev1.PullPolicy {
+	switch pullPolicy {
+	case "Always":
+		return corev1.PullAlways
+	case "Never":
+		return corev1.PullNever
+	case "IfNotPresent":
+		return corev1.PullIfNotPresent
+	default:
+		return corev1.PullAlways
```

Comment:
Can you change the default to be `PullIfNotPresent `?

Reply:
Ok